pkgs/applications/networking/instant-messengers/element/element-web.nix

```diff
@@ -34,6 +86,6 @@ in stdenv.mkDerivation rec {
     maintainers = lib.teams.matrix.members;
     license = lib.licenses.asl20;
     platforms = lib.platforms.all;
-    hydraPlatforms = [];
+    hydraPlatforms = [ ];
```

Comment:
If we're going to do building, I think we should have hydra build it (and it's unclear to me why this was put here in the first place).

Reply:
https://github.com/NixOS/nixpkgs/pull/28585#discussion_r135388315

Reply:
It sounds like it was there because in the past it was unnecessary since it was just copying the tarball?